server/bin/pbench-remote-sync-package-tarballs

```diff
@@ -34,6 +34,8 @@ esac
 
 satellite_config=$1
 output_tar_ball=$2
+tmp=$(dirname $(dirname ${output_tar_ball}))
```

Comment:
Not sure we can do this, too much knowledge of what the caller is doing in order for this to work safely in all cases.